### PR TITLE
#10266: Add documentation for relu_max and relu_min

### PR DESCRIPTION
**Pull request by mouliraj-mcw**
### Ticket
Link to Github Issue #10266 

### Problem description

Update doc with supported dtype and layout forrelu_min, rely_max op.

### What's changed
Document have been updated with supported data type and supported layout and sweeps test have been updated.

### Checklist
- [x] [All Post commit CI](https://github.com/tenstorrent/tt-metal/actions/runs/11345832332)
- [x] [Run sweeps for Relu max](https://github.com/tenstorrent/tt-metal/actions/runs/11345807948)
- [x] [Run sweeps for Relu min](https://github.com/tenstorrent/tt-metal/actions/runs/11345823265)

### CODE REVIEW

File: tests/sweep_framework/sweeps/eltwise/unary/relu_max/relu_max.py [hunk 1]
```diff
@@ -30,8 +30,8 @@
         "input_shape": gen_shapes([1, 1, 32, 32], [6, 12, 256, 256], [1, 1, 32, 32], 16)
         + gen_shapes([1, 32, 32], [12, 256, 256], [1, 32, 32], 16)
         + gen_shapes([32, 32], [256, 256], [32, 32], 32),
-        "input_a_dtype": [ttnn.bfloat16],
-        "input_a_layout": [ttnn.TILE_LAYOUT],
+        "input_a_dtype": [ttnn.bfloat16, ttnn.bfloat8_b],
+        "input_a_layout": [ttnn.TILE_LAYOUT, ttnn.ROW_MAJOR_LAYOUT],
         "input_a_memory_config": [ttnn.DRAM_MEMORY_CONFIG, ttnn.L1_MEMORY_CONFIG],
```
Comment (Aswinmcw) [inline tests/sweep_framework/sweeps/eltwise/unary/relu_max/relu_max.py:35]: if its single input, no need to mention as input_a

**Review by Aswinmcw — APPROVED** [review]
LGTM, just minor comment

File: tests/sweep_framework/sweeps/eltwise/unary/relu_min/relu_min.py [hunk 2]
```diff
@@ -31,7 +31,7 @@
         + gen_shapes([1, 32, 32], [12, 256, 256], [1, 32, 32], 16)
         + gen_shapes([32, 32], [256, 256], [32, 32], 32),
         "input_a_dtype": [ttnn.bfloat16],
```
Comment (Aswinmcw) [inline tests/sweep_framework/sweeps/eltwise/unary/relu_min/relu_min.py:33]: bf8?

**Review by VirdhatchaniKN — APPROVED** [review]

File: ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp [hunk 3]
```diff
@@ -1450,8 +1450,30 @@ void py_module(py::module& module) {
...
+           System memory is not supported.
+
+        )doc");
+    detail::bind_unary_operation_with_float_parameter(module, ttnn::relu_min, "lower_limit", "The min value for ReLU function", "This will carry out ReLU operation at min value instead of the standard 0",
+        R"doc(Supported dtypes, layouts, and ranks:
+
+           +----------------------------+---------------------------------+-------------------+
+           |     Dtypes                 |         Layouts                 |     Ranks         |
+           +----------------------------+---------------------------------+-------------------+
+           |    BFLOAT16, BFLOAT8_B     |          TILE                   |      2, 3, 4      |
```
Comment (KalaivaniMCW) [inline ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp:1471]: do we need to bf8_b if it is not included in sweep test ?

**Review by mcw-anasuya — APPROVED** [review]

**Review by jdesousa-TT — APPROVED** [review]
Please run these new relu sweeps in the sweeps pipeline before merging.

**Review by eyonland — APPROVED** [review]

**Review by eyonland — APPROVED** [review]

**Pull request merged**